.eslintrc.js

```diff
@@ -1,28 +1,13 @@
 module.exports = {
-  parser: '@typescript-eslint/parser',
-  extends: ['plugin:@typescript-eslint/recommended', 'plugin:react/recommended', 'react-app'],
-  plugins: ['@typescript-eslint', 'react'],
-  parserOptions: {
-    ecmaFeatures: {
-      jsx: true,
+  extends: ['plugin:@typescript-eslint/recommended'],
+  plugins: ['@typescript-eslint'],
+  overrides: [
+    {
+      files: ['*.test.ts'],
+      rules: {
+        '@typescript-eslint/no-var-requires': 0,
+        '@typescript-eslint/no-explicit-any': 0,
```

Comment:
in some cases, we need to explicitly provide a fake/bogus parameter to a function to test that it fails, that's why I set the eslint config to allow casting to the 'any' type in tests files